### ISSUE TITLE
Conda pack can't pack an environment that doesn't have python 

### ISSUE THREAD

**Issue by mariusvniekerk** [conda/conda-pack#33]
For example an r environment created as say

```
conda create -n renv r-base
```

**Comment by jcrist** [comment]
Yes, python is required for the `conda-unpack` command to work. The error message provided could be better though, currently it's:

```
$ conda-pack -n renv
Collecting packages...
CondaPackError: Unexpected failure, no version of python found in prefix '/Users/jcrist/anaconda/envs/renv'
```

Instead I think it should provide a nice explanation as to what the issue is (python isn't installed in the environment), and how the user can fix it (install python in the environment).

**Comment by mariusvniekerk** [comment]
Given that `conda-unpack` seems to use `#!/usr/bin/env python` as its shebang, do we need to be as prescriptive?

**Comment by jcrist** [comment]
This was fixed in #36. See https://github.com/conda/conda-pack/pull/36#issuecomment-413686675 for an example of packaging an R environment.
